win32ss/user/user32/windows/cursoricon.c

```diff
@@ -2015,6 +2015,12 @@ User32CallCopyImageFromKernel(PVOID Arguments, ULONG ArgumentLength)
 
 /************* PUBLIC FUNCTIONS *******************/
 
+#define COPYIMAGE_VALID_FLAGS ( \
```

Comment:
Did you verify that Windows does the same? Is this where 0x800 comes from?

Reply:
> Did you verify that Windows does the same?

Yes.

> Is this where 0x800 comes from?

Yes.

Reply:
An undocumented one for sure. It may be of interest to have something like:
```c
if (fuFlags & 0x800)
    ERR("CopyImage() using undocumented 0x800 flag\n");
```
in order to know which apps could be using it.
(Maybe some old Microsoft ones like Word, Excel 95 etc.)